css/src/components/messages.scss

```diff
@@ -0,0 +1,58 @@
+.messages {
+	display: flex;
+	flex-direction: column;
+	max-width: 100%;
+	gap: $layout-4;
+}
+
+.chat-bubble {
+	padding: $spacer-4;
+	border: $border-width solid $border !important;
```

Comment:
`!important` should only ever be used in components as a last resort

Reply:
removed